src/shogun/machine/KernelMachine.cpp

```diff
@@ -357,13 +357,11 @@ SGVector<float64_t> CKernelMachine::apply_get_outputs(CFeatures* data)
 				                  ? num_vectors
 				                  : (thread_num + 1) * step;
 
-#ifdef WIN32
 				for (int32_t vec = start; vec < end; vec++)
-#else
-				for (int32_t vec = start; vec < end && !cancel_computation();
-				     vec++)
-#endif
 				{
+#ifdef WIN32
```

Comment:
i'm not so sure if i get why do we have this #ifdef WIN32.... on non-windows we dont wanna allow controllers?

Reply:
We were thinking about removing them, since they seem to be not useful (we’ll see the outcome with the CIs).

Reply:
@shubham808 I guess we can then remove the `WIN32` guards.

Reply:
yes we can :)